user_guide/src/introduction.md

```diff
@@ -5,35 +5,34 @@
 This book is an introduction to the
 [`Polars` DataFrame library](https://github.com/pola-rs/polars). Its goal is to
 introduce you to `Polars` by going through examples and comparing it to other
-solutions. Some design choices are introduced here, and the optimal use of `Polars`
-described.
+solutions. Some design choices are introduced here. The guide will also introduce you to
+optimal usage of `Polars`.
 
 Even though `Polars` is completely written in [`Rust`](https://www.rust-lang.org/) (no
-runtime overhead!) and uses [`Arrow`](https://arrow.apache.org/) -the
-[native arrow2 `Rust` implementation](https://github.com/jorgecarleitao/arrow2)- as its foundation, the
+runtime overhead!) and uses [`Arrow`](https://arrow.apache.org/) -- the
+[native arrow2 `Rust` implementation](https://github.com/jorgecarleitao/arrow2) -- as its foundation, the
 examples presented in this guide will be mostly using its higher-level language
-bindings. The latter are merely a thin wrapper that will not offer more
-functionalities than the core library does.
+bindings. Higher-level bindings only serve as a thin wrapper for functionality implemented in the core library.
 
-For people used to [`Pandas`](https://pandas.pydata.org/), the
-[`Python`](https://www.python.org/) bindings are the easiest to get started with
-`Polars`, allowing easier experimentation.
+For [`Pandas`](https://pandas.pydata.org/) users, our
+[Python package](https://pypi.org/project/polars/) will offer the easiest way to get started with
+`Polars`.
 
 ## Goals and non-goals
 
-The goal of `Polars` is being a lightning fast DataFrame library that utilizes all
-available cores on your machine. Unlike tools like dask that tries to parallelize existing single-threaded libraries
-like `numpy` and `pandas`, `polars` is written from the ground up with parallelization of DataFrame queries in mind.
-It goes through great efforts to reduce redundant copies, traverse memory cache efficiently have minimal contention in
+The goal of `Polars` is to provide a lightning fast `DataFrame` library that utilizes all
+available cores on your machine. Unlike tools such as dask -- which tries to parallelize existing single-threaded libraries
+like `NumPy` and `Pandas` -- `Polars` is written from the ground up, designed for parallelization of queries on `DataFrame`s.
```

Comment:
I believe `like Numpy and Pandas` belong to the previous bullet point. This was a mistake.

Reply:
Pulling this up now, I'm not sure I follow this one. 
<img width="834" alt="image" src="https://user-images.githubusercontent.com/14341145/162623353-a2e3417a-538a-4483-b626-76f677071391.png">

This is one sentence. I don't see bullets. Maybe that line return makes it look like it's a bullet?

Reply:
Oh, my mistake. I saw the `-` minus signs from github (meaning removed lines) as bullet points. :smile: